android/KMEA/build.sh

```diff
@@ -74,14 +75,15 @@ if builder_start_action configure:engine; then
 
   # Copy KeymanWeb artifacts
   echo "Copying Keyman Web artifacts"
-  cp "$KEYMAN_WEB_ROOT/build/app/embed/$CONFIG/osk/ajax-loader.gif" "$ENGINE_ASSETS/ajax-loader.gif"
-  cp "$KEYMAN_WEB_ROOT/build/app/embed/$CONFIG/keyman.js" "$ENGINE_ASSETS/keymanandroid.js"
-  cp "$KEYMAN_WEB_ROOT/build/app/embed/$CONFIG/keyman.js.map" "$ENGINE_ASSETS/keyman.js.map"
-  cp "$KEYMAN_WEB_ROOT/build/app/embed/$CONFIG/osk/kmwosk.css" "$ENGINE_ASSETS/kmwosk.css"
-  cp "$KEYMAN_WEB_ROOT/build/app/embed/$CONFIG/osk/globe-hint.css" "$ENGINE_ASSETS/globe-hint.css"
-  cp "$KEYMAN_WEB_ROOT/build/app/embed/$CONFIG/osk/keymanweb-osk.ttf" "$ENGINE_ASSETS/keymanweb-osk.ttf"
-
-  cp "$KEYMAN_ROOT/common/web/sentry-manager/build/index.js" "$ENGINE_ASSETS/keyman-sentry.js"
+  cp "$KEYMAN_WEB_ROOT/build/app/webview/$CONFIG/index.js" "$ENGINE_ASSETS/keymanandroid.js"
+  cp "$KEYMAN_WEB_ROOT/build/app/webview/$CONFIG/index.js.map" "$ENGINE_ASSETS/keymanandroid.js.map"
```

Comment:
Okay, so I didn't realise that webview/$CONFIG/index.js was the final filename of KeymanWeb here. index.js is not a great output filename -- can we instead use something like keymanweb-webview.js (and then use the same name also in the assets folder for Keyman Engine for Android)?